src/pages/workspace/WorkspaceSettingsCurrencyPage.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useCallback} from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import useLocalize from '../../hooks/useLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import SelectionList from '../../components/SelectionList';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import withPolicy, {policyDefaultProps, policyPropTypes} from './withPolicy';
+import * as Policy from '../../libs/actions/Policy';
+
+const propTypes = {
+    /** Constant, list of available currencies */
+    currencyList: PropTypes.objectOf(
```

Comment:
Maybe it would be better to create a prop type for this, because the currency list elements have more properties than the symbol!

Reply:
@pac-guerreiro I get you, but it's a common pattern in the project to define only the fields you're consuming